.styleci.yml

```diff
@@ -2,4 +2,9 @@ preset: psr2
 
 enabled:
   - long_array_syntax
-  - duplicate_semicolon
\ No newline at end of file
+  - duplicate_semicolon
+
+finder:
+  exclude:
+    # exclude PHP7 illegal annotation classes to avoid false alert. those must not be attempted to be used in PHP 7
+    - "lib/Doctrine/ODM/PHPCR/Mapping/Annotations"
```

Comment:
when i list the explicit files, i still get complaints that the files are illegal